web-app/app/controllers/application_controller.rb

```diff
@@ -1,3 +1,3 @@
-class ApplicationController < ActionController::Base
+class ApplicationController < ActionController::Base.send(:include, MessagesModule)
```

Comment:
Why not just `include MessagesModule` in this class instead of sending this message to the superclass?